eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -155,8 +155,12 @@ jobs:
 
         - template: /eng/common/pipelines/templates/steps/verify-links.yml
           parameters:
-            Directory: ''
-            CheckLinkGuidance: $true
+            ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
Should $CheckLinkGuidance: $true apply to both if conditions?  What does linkguidance true do?

Reply:
Looks like the intention is just off on the CheckLinkGuidance, see https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/ci.yml#L242 fixing the indention will apply it to both cases.